invenio_app_ils/ui/src/invenio_app_ils/common/api/loan.js

```diff
@@ -3,10 +3,8 @@ import { toISO } from './date';
 import { DateTime } from 'luxon';
 
 const loanURL = '/circulation/loans/';
-
-const get = loanPid => {
-  return http.get(`${loanURL}${loanPid}`);
-};
+const get = loanPid => http.get(`${loanURL}${loanPid}`);
+const getPatchUrl = loanPid => `${loanURL}/${loanPid}/update`;
```

Comment:
What about if we get rid of the name `Patch`? not it is a POST.
`replaceItem`?

Reply:
I have refactored it in the circulation thing, and changed it to `LoanUpdate`, because thats what it does, and it could be used further, rather than just replacing the item. Your thoughts (and prayers) please.